.github/actions/update-csv-bundles/action.yml

```diff
@@ -0,0 +1,173 @@
+name: "Update CSV bundles"
+description: "Generate and finalize OLM CSV bundles for a given version"
+
+inputs:
+  version:
+    description: "Version without v-prefix"
+    required: true
+  token:
+    description: "GitHub token to create PRs"
+    required: true
+  changelog-path:
+    description: "Path to the changelog file"
+    required: false
+
+outputs:
+  major_minor:
+    description: "Major.minor part of the version"
+    value: ${{ steps.vars.outputs.major_minor }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Derive release branch name
+      id: vars
+      shell: bash
+      run: |
+        VERSION="${{ inputs.version }}"
```

Comment:
Could be moved to `env`, like here:

https://github.com/Dynatrace/dynatrace-operator/blob/6a71e3b2b5b4f7565a4f67f309e7f9dd2db3f408/.github/actions/update-csv-bundles/action.yml#L104-L105

Reply:
this could also be shared in the env between steps `echo VERSION=$VERSION >> $GITHUB_ENV`. make the extra `env` per step obsolete

Reply:
I will have a look